Quantum-Control-Applications/ESR/macros.py

```diff
@@ -0,0 +1,32 @@
+"""
```

Comment:
does this covers the case that you want to test the distance between waveforms further in time? like not the first two pulse but let's say the 8th and 10th?

Reply:
or in the case of CPMG were we check the 1st pulse of 2nd iteration vs the readout window?